extractor/test_wikipedia.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import json
+import ast
+from bs4 import BeautifulSoup, Tag, NavigableString   # XML(HTML)을 파싱
```

Comment:
Put all comments in English. :)

Reply:
Hahaha..... My mistake... :)
I'll fix it.